examples/updatecli.generic/yaml-files.yaml

```diff
@@ -0,0 +1,100 @@
+---
+title: Show a set of yaml resources as a generic example
```

Comment:
I'm late to the party, sorry.

But I thought the `examples/` folder was long deprecated (and removed) because its content is unmaintained.


I don't mind adding an example, for the sake of communication, but an end to end test is required then

Reply:
It's been too long ago, I can't remember why it's there 🤷 

I don't mind removing the example.

Reply:
I vaguely remember @olblak saying something about that. Does it ring a bell to you?

Should I just move this file to one of the e2e folders?

Reply:
Oh... Check the previous resolved conversation on this file: https://github.com/updatecli/updatecli/pull/650/files#r968050751

This example predates the e2e tests introdution.